README.md

```diff
@@ -1,11 +1,50 @@
-# documents
+# Dlang Graphics Workgroup
+
 
 The output of the Dlang Graphics Workgroup: a technical recommendation for an Image abstraction library, suitable to implement further graphics works.
 
-**Stage: requirements gathering.**
+
+**Step 1: requirements gathering.**
 
 Please vote on proposals here: https://github.com/DlangGraphicsWG/documents/pulls
 
 Current list of unsolved disagreements: https://github.com/DlangGraphicsWG/documents/wiki
 
 Current requirements gathered: [requirements.md](requirements.md)
+
+Current decision process: [decision-process.md](decision-process.md)
+
+
+# Goals
+
+
+The Dlang Graphics Workgroup is a collective of D library writers who want to
+make libraries together with a collegial design.
+
+We feel that Phobos leads the path towards building reusable, expressive abstractions;
+and that there are no reason a GUI Toolkit couldn't be _unopinionated_ to the same degree
+ std.range or std.allocator aren't.
```

Comment:
Better wording:
there is no reason a GUI Toolkit can't be unopinionated to the same degree std.range or std.allocator is.

Reply:
Thanks